automation/terraform/testnets/berkeley/main.tf

```diff
@@ -141,8 +143,8 @@ module "test-snapps" {
   snark_coordinators = [
     {
       snark_worker_replicas = 5
-      snark_worker_fee      = "0.025"
-      snark_worker_public_key = "B62qk4nuKn2U5kb4dnZiUwXeRNtP1LncekdAKddnd1Ze8cWZnjWpmMU"
+      snark_worker_fee      = "0.01"
+      snark_worker_public_key = "B62qmQsEHcsPUs5xdtHKjEmWqqhUPRSF2GNmdguqnNvpEZpKftPC69e"
```

Comment:
Is this public key in the ledger?

Reply:
yes, this was per deepthi's suggestion this morning.  this is the faucet i believe, the first account in the ledger.